doc/src/manual/parallel-computing.md

```diff
@@ -1266,10 +1272,12 @@ on the master process:
     it and returns the new cookie.
   * All connections are authenticated on both sides to ensure that only workers started by the master
     are allowed to connect to each other.
-  * The cookie must be passed to the workers at startup via argument `--worker <cookie>`. Custom ClusterManagers
-    can retrieve the cookie on the master by calling [`Base.cluster_cookie()`](@ref). Cluster managers
-    not using the default TCP/IP transport (and hence not specifying `--worker`) must call `init_worker(cookie, manager)`
-    with the same cookie as on the master.
+  * The cookie may be passed to the workers at startup via argument `--worker=<cookie>`. If argument
+    `--worker` is specified without the cookie, the worker tries to read the cookie from its
+    standard input (STDIN). The STDIN is closed immediately after the cookie is retrieved.
+  * ClusterManagers can retrieve the cookie on the master by calling [`Base.cluster_cookie()`](@ref).
```

Comment:
In fact other than the below
```
stdlib/parallel.md:Base.remotecall(::Any, ::Base.Distributed.AbstractWorkerPool, ::Any...)
stdlib/parallel.md:Base.remotecall_wait(::Any, ::Base.Distributed.AbstractWorkerPool, ::Any...)
stdlib/parallel.md:Base.remotecall_fetch(::Any, ::Base.Distributed.AbstractWorkerPool, ::Any...)
stdlib/parallel.md:Base.remote_do(::Any, ::Base.Distributed.AbstractWorkerPool, ::Any...)
```
which should be corrected, `Distributed` is not mentioned anywhere in the docs.